pkg/adapter/containers.go

```diff
@@ -437,7 +437,16 @@ func (r *LocalRuntime) Run(ctx context.Context, c *cliconfig.RunValues, exitCode
 			logrus.Errorf("Error removing container %s: %v", ctr.ID(), err)
 		}
 	}
-
+	if c.IsSet("genseccomp") {
```

Comment:
This isn't really a good place to put this... It only works with attached run, and only if you never detached. Safer place might be in `podman container cleanup` but then you might run into issues with containers with `--rm` set...